website/vcfa.erb

```diff
@@ -24,13 +24,16 @@
             <li<%= sidebar_current("docs-vcfa-data-source-tm-version") %>>
               <a href="/docs/providers/vcfa/d/tm_version.html">vcfa_tm_version</a>
             </li>
+            <li<%= sidebar_current("docs-vcfa-data-source-vcenter") %>>
+              <a href="/docs/providers/vcfa/d/vcenter.html">vcfa_vcenter</a>
+            </li>
           </ul>
         </li>
         <li<%= sidebar_current("docs-vcfa-resource") %>>
           <a href="#">Resources</a>
           <ul class="nav nav-visible">
-            <li<%= sidebar_current("docs-vcfa-resource-deleteme") %>>
-              <a href="/docs/providers/vcfa/r/deleteme.html">vcfa_placeholder</a>
```

Comment:
Should we delete this file? (`/docs/providers/vcfa/r/deleteme.html`)

Reply:
Deleted